### ISSUE TITLE
Using 0% workload on gantt diagram produces an infinite loop?

### ISSUE THREAD

**Issue by ggrossetie** [plantuml/plantuml#428]
When I submit the following diagram I get a 504 timeout:

```
@startgantt
title Plan

[UI 1] on {Bob} lasts 5 days
[UI 2] on {Bob:0%} lasts 4 days
[UI Task] on {Bob} {Alice} happens 1 days after [UI 2]'s end
[Start] -[dotted]-> [UI 1]
[UI 1] -> [UI 2]
[UI 2] -> [UI Task]
@endgantt
```

If I remove or update the 0% value then it's working fine.

**Comment by Mogztter** [comment]
I can confirm that there's an infinite loop here:

https://github.com/plantuml/plantuml/blob/099843c8307b4fc97d88f99a8fab3ee5a4b9a20c/src/net/sourceforge/plantuml/project/solver/SolverImpl.java#L52-L60

In this case `loadPlanable.getLoadAt(current)` always returns 0. As a result, `fullLoad` is never decremented and the while loop never ends.

I'm not sure how we should solve this issue. Should we throw a syntax error?

**Comment by arnaudroques** [comment]
Thanks for the report. This is fixed in last beta http://beta.plantuml.net/plantuml.jar
We now raise a syntax error in that case.
Tell us if you find other issues !

**Comment by Mogztter** [comment]
Thanks!
Where can I see the source code with the actual fix? It seems that this repository only contains a single commit when a new release is published.

**Comment by The-Lum** [comment]
Hi all,

_[This is an Issue Review]_
This is now fixed _(from ~v1.2021.0)_.
- Thanks @ggrossetie for the report.
- Thanks PlantUML team for your work.

Regards.
